src/components/Form/ErrorMessage.jsx

```diff
@@ -0,0 +1,20 @@
+import { Typography } from "@material-ui/core";
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme) => ({
+  error: {
+    margin: theme.spacing(2, 0, 1),
```

Comment:
In case of numbers it would be nice if we have them in constants like marginLeft, marginRight and so on.
It's more readable and easy to modify if someone has to.